### ISSUE TITLE
Google Colab TypeError: expected str, bytes or os.PathLike object, not NoneType

### ISSUE THREAD

**Issue by lenyabloko** [huggingface/transformers#10756]
## Environment info

- `transformers` version: 4.4.1
- Platform: Linux-4.19.112+-x86_64-with-Ubuntu-18.04-bionic
- Python version: 3.7.10
- PyTorch version (GPU?): 1.8.0+cu101 (True)
- Tensorflow version (GPU?): 2.4.1 (True)

The problem arises when using:

I started getting this error this morning without any changes on my side just loading my old Colab notebook (that worked few hours ago without any problem!)

The code that breaks is:
```
tokenizer = AutoTokenizer.from_pretrained('xlm-clm-ende-1024')


/usr/local/lib/python3.7/dist-packages/transformers/models/auto/tokenization_auto.py in from_pretrained(cls, pretrained_model_name_or_path, *inputs, **kwargs)
    417             else:
    418                 if tokenizer_class_py is not None:
--> 419                     return tokenizer_class_py.from_pretrained(pretrained_model_name_or_path, *inputs, **kwargs)
    420                 else:
    421                     raise ValueError(

/usr/local/lib/python3.7/dist-packages/transformers/tokenization_utils_base.py in from_pretrained(cls, pretrained_model_name_or_path, *init_inputs, **kwargs)
   1703 
   1704         return cls._from_pretrained(
-> 1705             resolved_vocab_files, pretrained_model_name_or_path, init_configuration, *init_inputs, **kwargs
   1706         )
   1707 

/usr/local/lib/python3.7/dist-packages/transformers/tokenization_utils_base.py in _from_pretrained(cls, resolved_vocab_files, pretrained_model_name_or_path, init_configuration, *init_inputs, **kwargs)
   1774         # Instantiate tokenizer.
   1775         try:
-> 1776             tokenizer = cls(*init_inputs, **init_kwargs)
   1777         except OSError:
   1778             raise OSError(

/usr/local/lib/python3.7/dist-packages/transformers/models/xlm/tokenization_xlm.py in __init__(self, vocab_file, merges_file, unk_token, bos_token, sep_token, pad_token, cls_token, mask_token, additional_special_tokens, lang2id, id2lang, do_lowercase_and_remove_accent, **kwargs)
    645             self.encoder = json.load(vocab_handle)
    646         self.decoder = {v: k for k, v in self.encoder.items()}
--> 647         with open(merges_file, encoding="utf-8") as merges_handle:
    648             merges = merges_handle.read().split("\n")[:-1]
    649         merges = [tuple(merge.split()[:2]) for merge in merges]

TypeError: expected str, bytes or os.PathLike object, not NoneType

```

## To reproduce

'''
!pip install transformers
!pip install pytorch-transformers
!pip install tensorboardX

'''

## Expected behavior

It all worked this morning without problem and for many month before that I have not touched the code.

**Comment by LysandreJik** [comment]
Hi @lenyabloko, indeed, there is an issue with the online repository of `xlm-clm-ende-1024`. @sgugger is currently fixing it right now. 

Thanks for letting us know, we'll let you know when it is fixed.

**Comment by LysandreJik** [comment]
It should be fixed now, thanks to @sgugger: [`huggingface#e824d7b`](https://huggingface.co/xlm-clm-ende-1024/tree/main)

**Comment by TatProg** [comment]
Same problem with 
`tokenizer = XLMTokenizer.from_pretrained('bert-base-multilingual-cased')`
